django_modern_rest/validation.py

```diff
@@ -277,24 +281,35 @@ def _validate_endpoints(
 
 
 @dataclasses.dataclass(slots=True, frozen=True, kw_only=True)
```

Comment:
```suggestion
```

It is not a dataclass by itself, it cannot be created, it is only a base class. Just add `__slots__`